packages/cascara/package.json

```diff
@@ -9,8 +9,8 @@
   },
   "license": "CC0-1.0",
   "sideEffects": false,
-  "main": "dist/cjs/index.js",
```

Comment:
Hi @brianespinosa: 
A test in docs is failing locally, this is the error:
![image](https://user-images.githubusercontent.com/87443054/150859992-0ca5b283-9496-4c3f-b6cf-6b56b8d266cb.png)

Should we get back this change in order to pass all the checkers? I wanted to create the PR anyway to point specifically to these lines.